inception-search-core/src/main/java/de/tudarmstadt/ukp/inception/search/SearchServiceImpl.java

```diff
@@ -259,6 +259,7 @@ public void afterAnnotationUpdate(AfterAnnotationUpdateEvent aEvent) throws Exce
 
         }
 
+        log.debug("Result Size : {}", results.size());
```

Comment:
Is this debug message still required? If yes, then `size` should not start with a capital letter.

Reply:
Removed. Not required